src/Development/IDE/LSP/CodeAction.hs

```diff
@@ -12,6 +12,7 @@ module Development.IDE.LSP.CodeAction
     ) where
 
 import           Language.Haskell.LSP.Types
+import Control.Arrow((&&&))
```

Comment:
I'd have used Data.Tuple.Extra, which provides the same function but specialised to tuples, since no one can understand arrows...